pkg/controller/kibana/pod.go

```diff
@@ -116,6 +122,20 @@ func NewPodTemplateSpec(ctx context.Context, client k8sclient.Client, kb kbv1.Ki
 		builder.WithVolumes(volume.Volume()).WithVolumeMounts(volume.VolumeMount())
 	}
 
+	// Kibana 7.5.0 and above support running with a read-only root filesystem,
+	// but require a temporary volume to be mounted at /tmp for some reporting features
+	// and a plugin volume mounted at /usr/share/kibana/plugins.
+	// Limiting to 7.10.0 here as there was a bug in previous versions causing rebuilding
+	// of browser bundles to happen on plugin install, which would attempt a write to the
+	// root filesystem on restart.
+	if v.GTE(version.From(7, 10, 0)) {
+		tmpVolume := volume.NewEmptyDirVolume("temp-volume", "/tmp")
```

Comment:
Why are you defining the temp volume inline in the `if` clause but the plugin volume is a global var? Can you change it to be consistent. Either approach is fine but use the same approach for both.

Reply:
Makes sense. I've moved all to be constants.